client/src/pages/AdminClientView.tsx

```diff
@@ -0,0 +1,93 @@
+import { gql, useQuery } from "@apollo/client";
+import React, { FunctionComponent, useEffect, useState } from "react";
+import { Spinner } from "react-bootstrap";
+import { useParams } from "react-router";
+
+import AdminPage from "../components/layouts/AdminPage";
+import Client from "../data/types/client";
+import ClientRequestTable from "../components/molecules/ClientRequestTable";
+import Container from "react-bootstrap/Container";
+import Request from "../data/types/request";
+import Row from "react-bootstrap/Row";
+
+
+interface ParamTypes {
+  id: string;
+}
+
+const AdminClientView: FunctionComponent = () => {
+  const { id } = useParams<ParamTypes>();
+  const [queryData, setQueryData] = useState<Request[]>([]);
+  const [clientName, setClientName] = useState<string | undefined>("");
+  const [openRequests, setOpenRequests] = useState(0);
+
+  const query = gql`
+    query ($c_id: ID) {
+      filterClients(filter: { id: $c_id }) {
+        fullName
+      }
+      filterRequests(filter: $c_id) {
+        requestId
+        quantity
+        dateCreated
+        fulfilled
+        deleted
+        client {
```

Comment:
Since we already know the client of the request (from the filter condition), do we need the graphql response to specify the client of each request?